realm/realm-library/src/main/java/io/realm/internal/Table.java

```diff
@@ -475,7 +469,7 @@ public long addEmptyRows(long rows) {
      * @return the row index of the appended row.
      */
     protected long add(Object... values) {
-        long rowIndex = addEmptyRow(false);
+        long rowIndex = addEmptyRow();
 
         checkImmutable();
```

Comment:
Though this is not related to the changes of PR, Is it OK to check immutability after adding a row?